bedrock/privacy/templates/privacy/index.html

```diff
@@ -81,6 +81,7 @@ <h1 itemprop="name">{{ doc.h1.string }}</h1>
         <li class="policy-websites"><a href="{{ url('privacy.notices.websites') }}">{{ _('Mozilla Websites, Communications &amp; Cookies') }}</a></li>
         <li class="policy-firefox"><a href="{{ url('privacy.notices.firefox') }}">{{ _('Firefox Browser') }}</a></li>
         <li class="policy-firefox-os"><a href="{{ url('privacy.notices.firefox-os') }}">{{ _('Firefox OS') }}</a></li>
+        <li class="policy-firefox-cliqz"><a href="{{ url('privacy.notices.firefox-cliqz') }}">{{ _('Firefox + Cliqz') }}</a></li>
```

Comment:
Turns out there is a german translation, but it's labelled incorrectly: https://github.com/mozilla/bedrock/pull/4946/files

I've asked in the bug if this is intentional or an error.